src/index.js

```diff
@@ -1,11 +1,14 @@
 const { highpoint: { dataURI } = {} } = self.window;
 
-const getBaseURI = () =>
-  dataURI ||
-  (document.baseURI || document.querySelector('base').href).replace(
-    /IScript_.*/,
-    'IScript_'
-  );
+const getBaseURI = () => {
```

Comment:
We are now going to reroute services based on the full service name. The getBaseURI function now replaces all but the last slug of the url with the dataURI if highpoint.dataURI is present.